RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -157,10 +157,24 @@ private extension CustomerCenterView {
                 }
             }
         }
+        .manageSubscriptionsSheet(isPresented: $viewModel.manageSubscriptionsSheet)
         .modifier(CustomerCenterActionViewModifier(actionWrapper: viewModel.actionWrapper))
+        .onCustomerCenterFeedbackSurveyCompleted { _ in
+            viewModel.manageSubscriptionsSheet = true
```

Comment:
why is this setting the `manageSubscriptionsSheet`? the modifier is for `onCustomerCenterFeedbackSurveyCompleted`

Reply:
Also we are never setting if back to `false`

Reply:
> why is this setting the manageSubscriptionsSheet? the modifier is for onCustomerCenterFeedbackSurveyCompleted

This gets called when you tap the feedback survey, after cancelling. Before we were using `showManageSubscriptions`, but I've replaced that with the SwiftUI approach.

It gets back to false by SwiftUI.

Reply:
ohh I see. The thing is that any path can have a feedback survey completed, not only the cancel or change. For example, a custom URL could also have a feedback survey attached to it. This means the manage subscription sheet would be displayed after completing ANY feedback survey

Reply:
As discussed, I've removed this. I don't remember why I added it, but it's definitely not needed to cover this 😅